### DISCUSSION TITLE
Problem that graph is not drawn when using string x of coordinate chart

### DISCUSSION THREAD

**Discussion by hoonick91** [nhn/tui.chart#638] [Q&A]
안녕하세요 toast ui를 사용하고 있는 개발자 입니다.

Basic Chart를 사용하기에는 이슈가 있어 Coordinate Chart를 사용하려고 합니다.

하지만 

```
const data = {
  series: [
    {
      name: 'SiteA',
      data: [
        ["1", 202],
        ["7", 350],
        ["8", 213],
        ["9", 230],
        ["12", 230],
      ],
    }
  ],
};
```

위와 같이 데이터가 들어갈 경우 그래프가 그려지지않아 혹시 다른 방법이 있는지 문의 드립니다.

감사합니다.

**Comment by jung-han** [comment]
@hoonick91 
In the coordinate chart, x values can only be `numbers` and `date` values.
In this case, it will be solved by changing the x value to a number and set it in.
